### PR TITLE
Feature/lt 844 add additional documents

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: templates/apply_for_a_licence/additional_documents/additional_documents.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		<h1 class="govuk-heading-l">{% get_string 'additional_documents.title'%}</h1>
+	</div>
+	<div class="lite-app-bar__controls">
+		<a href="{% url attach_document_link draft_id %}" class="govuk-button" draggable="false" role="button">
+			{% get_string 'additional_documents.add' %}
+		</a>
+	</div>
+</div>
+
+{% if additional_documents|length %}
```
Comment (BenAndrew-Kainos) [inline templates/apply_for_a_licence/additional_documents/additional_documents.html]: No |length required

File: templates/apply_for_a_licence/additional_documents/additional_documents.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                        Download
+                    </a>
+                    <br/>
+                    <a href="{% url delete_document_link draft_id additional_document.id %}" id="document_delete" class='govuk-link govuk-link--no-visited-state'>
+                        Delete
+                    </a>
+                {% else %}
+                    Processing
+                {% endif %}
+		</tr>
```
Comment (BenAndrew-Kainos) [inline templates/apply_for_a_licence/additional_documents/additional_documents.html]: Missing </td>

File: templates/apply_for_a_licence/additional_documents/additional_documents.html (hunk 1 above)
Comment (stephanieboyd) [reply templates/apply_for_a_licence/additional_documents/additional_documents.html]: Removed

File: apply_for_a_licence/urls.py [hunk 3]
```diff
@@ -83,5 +84,10 @@
     # ex: /apply_for_a_licence/<uuid:pk>/ultimate-end-user/download-document/ - Delete a document
     path('<uuid:pk>/third-parties/<uuid:tp_pk>/document/delete', documents.DeleteDocument.as_view(), name="third_party_delete_document"),
     # ex: /<uuid:pk>/third-parties/remove
-    path('<uuid:pk>/third-parties/<uuid:ueu_pk>/remove', third_parties.RemoveThirdParty.as_view(), name='remove_third_party')
+    path('<uuid:pk>/third-parties/<uuid:ueu_pk>/remove', third_parties.RemoveThirdParty.as_view(), name='remove_third_party'),
+
```
Comment (andytaylorkainos) [inline apply_for_a_licence/urls.py]: whats with the blank line?

File: conf/constants.py [hunk 4]
```diff
@@ -22,6 +22,7 @@
 AUTHENTICATION_URL = USERS_URL + 'authenticate/'
 CASE_NOTES_URL = '/case-notes/'
 ECJU_QUERIES_URL = '/ecju-queries/'
+ADDITIONAL_DOCUMENT_URL = '/documents/'
```
Comment (andytaylorkainos) [inline conf/constants.py]: should this not just be `DOCUMENTS_URL`?

File: apply_for_a_licence/forms/end_user.py [hunk 5]
```diff
@@ -43,10 +43,18 @@ def new_end_user_forms():
     return FormGroup(third_parties_standard_form('Who will be the final recipient (end-user) of your goods?'))
 
 
-def attach_document_form(draft_url, title, back_text, return_later_text):
+def attach_document_form(draft_url, title, back_text, return_later_text, description_text, description=False):
```
Comment (a-bek) [inline apply_for_a_licence/forms/end_user.py]: You should be able to do your conditional check by adding just one new parameter: 

`description_text=None`, and make the conditional below `if description_text:`

File: apply_for_a_licence/urls.py (hunk 3 above)
Comment (stephanieboyd) [reply apply_for_a_licence/urls.py]: All the urls in this file are grouped and separated by a newline, e.g. third parties, ultimate end users

File: conf/constants.py (hunk 4 above)
Comment (stephanieboyd) [reply conf/constants.py]: yeah, changed it

File: apply_for_a_licence/forms/end_user.py (hunk 5 above)
Comment (a-bek) [inline apply_for_a_licence/forms/end_user.py]: You should be able to do your conditional check by adding just one new parameter:

Setting the default `description_text=None`, and make the conditional below `if description_text:`

Comment (stephanieboyd) [reply apply_for_a_licence/forms/end_user.py]: Good spot, changed this up

File: conf/constants.py (hunk 4 above)
Comment (stephanieboyd) [reply conf/constants.py]: Naw, on another look - it's plural documents for additional documents

File: apply_for_a_licence/urls.py [hunk 6]
```diff
@@ -83,5 +84,10 @@
...
+    path('<uuid:pk>/additional-documents/', apply_for_a_licence.views.additional_documents.AdditionalDocuments.as_view(), name='additional_documents'),
+    path('<uuid:pk>/additional-document/attach', documents.AttachDocuments.as_view(), name='attach_additional_document'),
```
Comment (Mark-Spiers-kainos) [inline apply_for_a_licence/urls.py]: can you add comments for example paths?

**Pull request merged**